core/Tracker/Request.php

```diff
@@ -341,9 +366,18 @@ public function getParam($name)
         $paramDefaultValue = $supportedParams[$name][0];
         $paramType = $supportedParams[$name][1];
 
-        $value = Common::getRequestVar($name, $paramDefaultValue, $paramType, $this->params);
+        if ($this->hasParam($name)) {
```

Comment:
`Common:getRequestVar` takes a load of time even when just the default value is used. Therefore we make this much faster here.